link.go

```diff
@@ -194,7 +195,94 @@ func linkEntities(ctx *ls.Context, tx neo4j.Transaction, config Config, entityRo
 	return nil
 }
 
-func linkToThisEntity(ID []string) error {
+// ID is entity id of entity root
+func linkToThisEntity(ctx *ls.Context, tx neo4j.Transaction, config Config, entityRoot *lpg.Node, ID []string, nodeMap map[*lpg.Node]uint64) error {
 	// TODO: Search for all nodes with Ref to this entity, and link them
+	vars := make(map[string]interface{})
+	_, ok := entityRoot.GetProperty(ls.EntitySchemaTerm)
+	if !ok {
+		return errors.New("invalid schema node, given entity root must contain schema node id property")
+	}
+	var fkNodesRec neo4j.Result
+	var err error
+	entityID, _ := entityRoot.GetProperty(ls.SchemaNodeIDTerm)
+	if len(ID) > 1 {
+		// fkNodesRec, err = tx.Run(fmt.Sprintf("MATCH (n {`%s`: `%s`}) WHERE n.`%s` IN $ids return n", ls.ReferenceFKFor, schemaNodeId.(*ls.PropertyValue).AsString(), ls.ReferenceFK), map[string]interface{}{"ids": ID})
+		fkNodesRec, err = tx.Run(fmt.Sprintf(
+			"MATCH (n) WHERE n.`%s`= %s AND n.`%s` IN $ids RETURN n",
+			ls.ReferenceFKFor, quoteStringLiteral(entityID.(*ls.PropertyValue).AsString()), ls.ReferenceFK),
```

Comment:
Use config to map the ls.ReferenceFKFor and ReferenceFK terms. That may be shortened.